src/components/AudioPlayer/AudioPlayer.web.js

```diff
@@ -39,24 +39,41 @@ export default class AudioPlayerSub extends Component {
 
   componentDidUpdate(prevProps) {
     const {
-      track: { url },
+      track: { url, title: title },
       playing,
       progress,
-      endSong,
+      updatePlaying,
+      topScreen,
```

Comment:
```suggestion
      topScreen,
      endSong,
```
It looks like you accidentally deleted this variable

Reply:
I added another object destructuring if the endSong condition is met on line 64